docs/client.rst

```diff
@@ -1,1078 +1,325 @@
+          :code:`delete_session` will perform until receiving a status message.
+        - :code:`<number>`
+        - :code:`10`
+    *   - :code:`UCLIENT_MIN_SESSION_CONNECTION_INTERVAL`
+        - This value represents how long it will take to send a new :code:`create_session` |br|
+          or :code:`delete_session` if the first attempt was left answered.
+        - :code:`<number>`
+        - :code:`1000`
+    *   - :code:`UCLIENT_MIN_HEARTBEAT_TIME_INTERVAL`
+        - In a reliable communication, this value represents how long it will take for |br|
+          the first heartbeat to be sent. The wait time for the next heartbeat will be |br|
+          double. It is measured in milliseconds.
+        - :code:`<number>`
+        - :code:`1`
+    *   - :code:`UCLIENT_BIG_ENDIANNESS`
+        - This value must correspond to the memory endianness of the device in |br|
+          which the *Client* is running. :code:`OFF` implies that the machine is little-endian |br|
+          and :code:`ON` implies big-endian.
+        - :code:`<bool>`
+        - :code:`OFF`
+    *   - :code:`UCLIENT_UDP_TRANSPORT_MTU`
+        - This value corresponds to the *Maximum Transmission Unit (MTU)* that can |br|
+          be sent and/or received by UDP. It is measured in bytes and, internally, it |br|
+          corresponds to the creation of a buffer this size.
+        - :code:`<number>`
+        - :code:`512`
+    *   - :code:`UCLIENT_TCP_TRANSPORT_MTU`
+        - This value corresponds to the *Maximum Transmission Unit (MTU)* that can |br|
+          be sent and/or received by TCP. It is measured in bytes and, internally, it |br|
+          corresponds to the creation of a buffer this size.
+        - :code:`<number>`
+        - :code:`512`
+    *   - :code:`UCLIENT_SERIAL_TRANSPORT_MTU`
+        - This value corresponds to the *Maximum Transmission Unit (MTU)* that can |br|
+          be sent and/or received by Serial. It is measured in bytes and, internally, it |br|
+          corresponds to the creation of a buffer this size.
+        - :code:`<number>`
+        - :code:`512`
+
+.. _read_access:
+
+Read Access Delivery Control
+----------------------------
+
+The Read Access Delivery Control handles the read operation from a *datareader* previously created
+on the *Agent* to fetch data from the middleware.
+It comes with an optional ``control`` argument, that allows the *Client* setting the following parameters:
+
+* ``max_bytes_per_second``: Maximum rate at which data messages may be returned, measured in bytes per secpond.
+* ``max_elapsed_time``: Maximum amount of time that can be spent by the Agent in delivering the topic, measured in seconds.
+* ``max_samples``: Maximum number of topics that the Agent can send to the Client.
+* ``min_pace_period``: Minimum elapsed time between two topics deliveries, measured in milliseconds,.
+
+For more information, consult the :ref:`read_access_api` of the :ref:`client_api_label`.
+
+Creation Mode
```

Comment:
For me, it is very confusing that this section is called "Creation mode" and the one right after is called "Creation Mode table", yet they verse about very different things. We should come up with a clearer way to re-structure this